packages/ui/src/common/api/schemas/schema.graphql

```diff
@@ -817,6 +822,7 @@ enum ApplicationFormQuestionType {
 }
 
 type ApplicationFormQuestion {
+  id: ID!
```

Comment:
Added this ID so that I could seed the mocked data more easily, but I see that it's still missing in the schema in the newest Olympia PR. Not sure why some types don't have IDs

Reply:
Maybe not all of them are required for the Hydra to create GraphQL schema. The one in the PR is later transformed by Hydra to the GraphQL format. My second guess is that only specific IDs might need to be defined.